binaryen-sys/Shim.cpp

```diff
@@ -8,17 +8,20 @@
 #include "tools/fuzzing.h"
 #include "binaryen-c.h"
 
+#include "wasm.h"           // For Feature enum
+#include "wasm-validator.h" // For WasmValidator
+
 using namespace wasm;
 
-extern "C" BinaryenModuleRef translateToFuzz(const char *data, size_t len) {
+extern "C" BinaryenModuleRef translateToFuzz(const char *data, size_t len, bool emitAtomics = true) {
```

Comment:
I feel a little wary about making this param with a default value simply because I don't understand how `extern "C"` works with arguments with default values. At the same time default value isn't needed (right?) here since this function is only supposed to be called via FFI. 
With all that, I think that it would be better to just remove the default value from the argument.

Reply:
Good point -- removed